.gitignore

```diff
@@ -18,6 +18,7 @@ cupy.egg-info/
 dist/
 htmlcov/
 .idea/
+.vscode/
```

Comment:
We do not have `.gitignore` setting for development environments. Would you remove this change from the PR? Instead, you can use `~/.config/git/ignore` to ignore vscode files in all situations.